src/containers/MultidisciplinarySubject/components/MultidisciplinaryTopic.jsx

```diff
@@ -8,15 +8,21 @@
 import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
Denne filen er nesten identisk med `src/containers/SubjectPage/components/Topic.tsx`. Kunne dette vært redusert til en komponent?

Reply:
Eg kopierte inn kode fra Topic for å løse problemet. Burde sjekke eksempelkoden i designmanualen for å sjå om den kan slås sammen med Topic.